### DISCUSSION TITLE
Migration from TIBCO BPM to CAMUNDA 8 sizing requirement for CAMUNDA 8 Installation

### DISCUSSION THREAD

**Discussion by Sreenivas1209** [camunda-community-hub/community#163] [Q&A]
HI, We are doing migration from TIBCO BPM to CAMUNDA 8 Self Managed. As part of this we have installed Kubernates in Redhat Linux box with 1 master node with 2 worker nodes. on top of this I am installing CAMUNDA 8. 
Master and worker nodes have memories like below after Kubernates and CAMUNDA 8 Installation.
**Master Node as below - 
#df -h /**
Filesystem                        Size   Used        Avail    Use%   Mounted on
/dev/mapper/vg00-root  4.0G  104M       3.9G     3%       /
**#free -h**
               total        used        free      shared      buff/cache   available
Mem:      7.5Gi       2.2Gi       973Mi     330Mi       4.4Gi          4.7Gi
Swap:       0B          0B           0B

**1st Worker Node as below -**
# df -h /
Filesystem                        Size    Used    Avail      Use%    Mounted on
/dev/mapper/vg00-root  4.0G   95M     3.9G       3%              /
# free -h
                    total        used        free        shared   buff/cache   available
Mem:          5.5Gi       2.3Gi       879Mi       279Mi       2.4Gi       2.7Gi
Swap:            0B          0B          0B

**2nd Worker Node as below -** 
# df -h /
Filesystem                        Size    Used    Avail     Use%    Mounted on
/dev/mapper/vg00-root  4.0G   95M     3.9G      3%       /
# free -h
                  total        used        free      shared  buff/cache   available
Mem:          5.5Gi       2.0Gi       445Mi       286Mi       3.1Gi       3.0Gi
Swap:            0B          0B          0B

**Issue**
After installing Camunda all pods of CAMUNDA is not getting up and running.

**Ealstic search pods are not getting to ready state due to below issues
0/3 nodes are available: 1 node(s) had untolerated taint {node-role.kubernetes.io/control-plane: }, 2 Insufficient cpu. preemption: 0/3 nodes are available: 1 Preemption is not helpful for scheduling, 2 No preemption victims found for incoming pod.**

so i made master node also to work as worker node.. after making this change i dont see "1 node(s) had untolerated taint {node-role.kubernetes.io/control-plane" issue.. but still am getting **"Insufficient cpu. preemption: 0/3 nodes are available: 1 Preemption is not helpful for scheduling, 2 No preemption victims found for incoming pod"**,

Can someonehelp what is the minimum sizing and cpus/RAM/memories required for Camunda 8 self managed with Kubernates setup in redhat linux box required? please node that its for dev environment..

**Comment by xomiamoore** [comment]
Hey @Sreenivas1209, this GitHub Discussions board is for questions related to the Camunda Community Hub and the community-led projects here -- [the Camunda forum](http://forum.camunda.io/) is your best bet for questions like this. Thanks!
